pycgm/axis.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#pyCGM
+
+# Copyright (c) 2015 Mathew Schwartz <umcadop@gmail.com>
+# Core Developers: Seungeun Yeon, Mathew Schwartz
+# Contributors Filipe Alves Caixeta, Robert Van-wesep
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#pyCGM
+
+"""
+This file is used in joint angle and joint center calculations.
+"""
+
+import math
+import numpy as np
+
+def find_joint_center(mark_a, mark_b, mark_c, delta):
+    """Calculate the Joint Center.
+    This function is based on the physical markers mark_a, mark_b, mark_c
+    and the resulting joint center are all on the same plane.
+    Parameters

```

Comment:
Did a line break do something to your family, or can we forgive it and start using them :)